r2_chatterbot/util/command_type.py

```diff
@@ -0,0 +1,165 @@
+import string
+import time
+from quantulum3 import parser
+
+import nlp_util
+import path_planning
+
+# Facial Recognition
+def isFaceRecognition(text):
```

Comment:
can we import this function from the facial rec file? or remove it from face_recognition.py if we're not using it here